### PR TITLE
v1.20 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+ - Channel sidebar now remains open at a fixed width on tablet devices.
+ 
+#### iOS Keyboard Dismissal
+ - If the keyboard is open, swiping down past it now closes it.
+
+### Improvements
+ - Increased the double tap delay for post action buttons.
+ - Implemented assets for Adaptive icons.
+ - Users are now brought to the bottom of the channel when posting a message.
+
```
Comment (enahum) [inline CHANGELOG.md]: lets add "Added support on iOS for IPv6 on LTE networks"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### Tablet Improvements
+ - Channel sidebar now remains open at a fixed width on tablet devices.
+ 
+#### iOS Keyboard Dismissal
+ - If the keyboard is open, swiping down past it now closes it.
+
```
Comment (enahum) [inline CHANGELOG.md]: Add profile telemetry for Android beta builds?

File: CHANGELOG.md (hunk 1 above)
Comment (enahum) [reply CHANGELOG.md]: Support for group constraints with 5.12 servers ?

Comment (esethna) [reply CHANGELOG.md]: Also "Users can now execute actions while the keyboard is open"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+ - Fixed an issue on Android where the "Write to [channel name]" was cut off for group messages with several users.
+ - Fixed an issue where users were not able to flag or unflag posts in a read-only channel.
+ - Fixed an issue where the progress indicator was negative while downloading a video.
+ - Fixed an issue where the edit post modal didn’t have an autocorrect.
+ - Fixed an issue where the 'I forgot my password' option was available on the mobile client even with Email Authentication disabled on the server.
+ - Fixed an issue with large separation between placeholders on iPad when a channel was loading.
+ - Fixed an issue where "Show More" was not removed after the post was edited to a single line.
+ 
+### Known Issues
+ - Buttons inside ephemeral posts are not clickable / functional on the mobile app.
```
Comment (esethna) [inline CHANGELOG.md]: Can you provide a link to the ticket in the comments here?

**Comment by amyblais** [comment]
@enahum Can you help merge this at some point today?

**Pull request merged**